src/app/users/confirmNewUser.js

```diff
@@ -187,13 +200,17 @@ const post = async (req, res) => {
         await addUserService(uid, service.serviceId, organisationId, service.roles, req.id);
 
         await notificationClient.sendServiceRequestApproved(
-          req.session.user.email, 
-          req.session.user.firstName, 
-          req.session.user.lastName, 
+          req.session.user.email,
+          req.session.user.firstName,
+          req.session.user.lastName,
           org,
           serviceDetails.name,
-          roleDetails.map(i => i.name)
-        )
+          roleDetails.map((i) => i.name),
+          {
+            id: mngUserOrganisationDetails.role.id,
```

Comment:
this object can be stored in a constant an used everywhere where notificationClient.sendServiceRequestApproved() is called

Reply:
done